dashboard/app/models/script_level.rb

```diff
@@ -219,7 +219,7 @@ def icon
     end
   end
 
-  def summarize
+  def summarize(include_prev_next=true)
```

Comment:
It's unfortunately not at all clear to me where they might be used; I am only confident that they are _not_ used in the script overview.

I can in a future PR investigate removing them entirely.